### DISCUSSION TITLE
Home Assistant running in Docker

### DISCUSSION THREAD

**Discussion by mr-manuel** [blakeblackshear/frigate#1925] [Ask A Question] [answered]
Hello,

I'm running Frigate in Docker and also Home Assistant. Is there a way to add Frigate to Home Assistant when both are running in Docker? Unfortunately I only found installation guides when Home Assistant is installed natively on Linux and the Supervisor menu is active.

**Comment by blakeblackshear** [comment, marked as answer]
You can use the integration and lovelace card to integrate most of what you need in home assistant. I suggest adding frigate to a reverse proxy with authentication and exposing it on it's own subdomain.

**Reply by mr-manuel** [reply]
Thanks!
